build.gradle

```diff
@@ -123,7 +123,7 @@ check.dependsOn installDist
 
 dependencies {
     compile files("${System.properties['java.home']}/../lib/tools.jar")
-    compile 'org.broadinstitute:gatk:4.alpha.1-104-g213595d-20160606.150121-2'
+    compile 'org.broadinstitute:gatk:4.alpha.1-115-gc143179-20160610.175138-1'
```

Comment:
this is a snapshot off of the build in public, it will have to be updated before merging with what actually goes into public